WooCommerce/Classes/Analytics/WooAnalyticsEvent.swift

```diff
@@ -2486,10 +2486,12 @@ extension WooAnalyticsEvent {
             case dashboardMainStats
             case analyticsHub
             case appStartup
+            case pointOfSaleLoaded
         }
 
         private enum Keys {
             static let waitingTime = "waiting_time"
+            static let milliseconds_time_elapsed = "milliseconds_time_elapsed_in_splash_screen"
```

Comment:
nit: as the value is pretty specific to "splash screen", probably want to name the key accordingly. Also would be great to keep it camel case.

Reply:
True that, updated: https://github.com/woocommerce/woocommerce-ios/pull/15047/commits/47924eac8e1d61894ada8234adadc5b42db9e48b